drivers/regulator/regulator_da1469x.c

```diff
@@ -281,6 +282,19 @@ static int regulator_da1469x_set_voltage(const struct device *dev, int32_t min_u
 	uint16_t idx;
 	uint32_t mask;
 
+#define PLL_FREQ	96000000
```

Comment:
this should be better obtained from device tree using macros

Reply:
Done